src/EnergyPlus/Autosizing/SystemAirFlowSizing.cc

```diff
@@ -777,13 +777,38 @@ Real64 SystemAirFlowSizer::size(EnergyPlusData &state, Real64 _originalValue, bo
                     }
 
                 } else {
-                    auto &sysSizPeakDDNum = state.dataSize->SysSizPeakDDNum(this->curSysNum);
-
-                    if (this->airLoopSysFlag) {
-                        auto const &unitarysysEqSizing = this->unitarySysEqSizing(this->curSysNum);
-                        if (unitarysysEqSizing.CoolingAirFlow && unitarysysEqSizing.HeatingAirFlow) {
-                            this->autoSizedValue = std::max(unitarysysEqSizing.CoolingAirVolFlow, unitarysysEqSizing.HeatingAirVolFlow);
-                            if (this->autoSizedValue == unitarysysEqSizing.CoolingAirVolFlow) {
+                    if (this->curOASysNum > 0) {
```

Comment:
Feels a little weird to size the OA system fan using the air loop flow rate (instead of using DesOutAirVolFlow) but that's what happened before and is how the MaxOA gets sized for the OA controller.